lua/octo/completion.lua

```diff
@@ -11,12 +11,13 @@ local M = {}
 local function get_url(url, params)
   url = url .. "?foo=bar"
   for k, v in pairs(params) do
-    url = format("%s\\&%s=%s", url, k, v)
```

Comment:
dont remember why I was escaping the `&`, probably because before using `gh` I was using a custom `curl` wrapper. thanks for fixing it!